src/components/review/StorageReview.jsx

```diff
@@ -52,10 +54,30 @@ import "./StorageReview.scss";
 
 const _ = cockpit.gettext;
 const idPrefix = "storage-review";
+const SCREEN_ID = "anaconda-screen-review";
 
-export const StorageReview = () => {
+export const StorageReview = ({ setStepNotification }) => {
     const { diskSelection } = useContext(StorageContext);
     const selectedDisks = diskSelection.selectedDisks;
+    const freeSpace = useFreeSystemMountPointsSpace();
+    const requiredSize = useRequiredSize();
+
+    useEffect(() => {
+        if (requiredSize > freeSpace && setStepNotification) {
+            const msg = cockpit.format(
+                _("Not enough free space on assigned mount points. Required: $0 Available: $1."),
+                cockpit.format_bytes(requiredSize),
+                cockpit.format_bytes(freeSpace)
+            );
+            setStepNotification({
+                message: msg,
+                step: SCREEN_ID,
+                variant: "warning"
+            });
+        } else if (setStepNotification) {
```

Comment:
I would prefer the setStepNotification check in the start if the useEffect:

if (setStepNotification === undefined) { return };